lsp-completion.el

```diff
@@ -521,7 +521,7 @@ Others: TRIGGER-CHARS CANDIDATES"
                          (lsp:command-command command?)
                          err))))
 
-        (when (and lsp-signature-auto-activate
+        (when (and (memq :after-completion lsp-signature-auto-activate)
```

Comment:
We should handle `lsp-signagure-auto-active` being `t` and/or document this as a breaking change.

Reply:
Done. Handle `lsp-signature-auto-active` being `t`